app/src/main/java/io/rapidpro/androidchannel/RapidProFirebaseMessageService.java

```diff
@@ -0,0 +1,33 @@
+package io.rapidpro.androidchannel;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import java.util.Map;
+
+
+public class RapidProFirebaseMessageService extends FirebaseMessagingService {
```

Comment:
I'd disregard my comment, I don't see a strong reason to go renaming everything else.